DFImageManager/Source/Core/Caching/DFImageCache.m

```diff
@@ -52,7 +55,7 @@ - (instancetype)init {
 - (nullable DFCachedImageResponse *)cachedImageResponseForKey:(nullable id<NSCopying>)key {
     DFCachedImageResponse *response = [_cache objectForKey:key];
     if (response) {
-        if (response.expirationDate > CACurrentMediaTime()) {
+        if (response.expirationDate > mach_absolute_time()) {
```

Comment:
You can't use mach_absolute_time() like that. It returns time in processing units, not seconds like CACurrentMediaTime(). Conversion between mach_absolute_time() units and seconds is quite complicated.

Anyway, both functions are useful when you need to measure time in terms of nanoseconds, which is not the case here. NSDate looks like a good candidate, but it's rather heavy and this code needs to run on the main thread. I would rather use something that is faster.